priv/gettext/default.pot

```diff
@@ -227,21 +188,107 @@ msgid "Name"
 msgstr ""
 
 #, elixir-format
-#: lib/companies_web/templates/company/form.html.eex:51
+#: lib/companies_web/templates/company/form.html.eex:54
 msgid "Optional fields"
 msgstr ""
 
 #, elixir-format
-#: lib/companies_web/templates/job/form.html.eex:17
+#: lib/companies_web/templates/job/index.html.eex:36
 msgid "Title"
 msgstr ""
 
 #, elixir-format
-#: lib/companies_web/templates/job/form.html.eex:25
+#: lib/companies_web/templates/job/form.html.eex:23
 msgid "Url"
 msgstr ""
 
 #, elixir-format
-#: lib/companies_web/templates/company/form.html.eex:34
+#: lib/companies_web/templates/company/form.html.eex:37
 msgid "Website"
 msgstr ""
+
+#, elixir-format
+#: lib/companies_web/views/admin/pending_change_view.ex:11
+msgid "Approved Change"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/admin/pending_change/index.html.eex:18
+#: lib/companies_web/views/admin/pending_change_view.ex:14
+msgid "Approved Changes"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/company/recent.html.eex:46
+msgid "Browse full list"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/company/index.html.eex:4
+#: lib/companies_web/templates/layout/navbar.html.eex:18
+msgid "Companies"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/job/form.html.eex:15
+msgid "Job Title"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/job/index.html.eex:3
+#: lib/companies_web/templates/layout/navbar.html.eex:19
+msgid "Jobs"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/job/index.html.eex:38
+msgid "Last update"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/views/admin/pending_change_view.ex:10
+msgid "Pending Change"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/admin/pending_change/index.html.eex:13
+#: lib/companies_web/views/admin/pending_change_view.ex:16
+msgid "Pending Changes"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/views/admin/pending_change_view.ex:12
+msgid "Rejected Change"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/admin/pending_change/index.html.eex:23
+#: lib/companies_web/views/admin/pending_change_view.ex:15
+msgid "Rejected Changes"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/job/index.html.eex:35
+msgid "Remote"
+msgstr ""
+
+#, elixir-format
+#: lib/companies_web/templates/job/show.html.eex:16
+msgid "Remote:"
```

Comment:
@gemantzu can you find where we use this and change it so it's just `Remote`, that way we don't have 2 nearly identical translations